inc/init.php

```diff
@@ -17,6 +17,11 @@
 require $conf . 'locales.inc.php';
 require $conf . 'sources.inc.php';
 
+// Override sources for functional tests
+if (getenv('TRAVIS')) {
```

Comment:
That seems a bad approach, we need to get these tests to also work locally, not just on Travis. Generally speaking, testing for the structure of the data returned instead of the exact content which varies over time seems more reliable.